{{cookiecutter.project_slug}}/clients/vue3/src/views/Login.vue

```diff
@@ -1,27 +1,26 @@
 <template>
-  <div class="login">
-    <form @submit.prevent="attemptLogin()">
+  <div class="login container mx-auto content-center">
+    <h1 class="login__header text-2xl font-bold mb-6">Log In.</h1>
+    <form class="login__form flex flex-col items-center" @submit.prevent="attemptLogin()">
       <InputField
-        v-model:value="form.email.value"
+        v-model="form.email.value"
```

Comment:
**Info:** Don't need the `:value` here (and it gets flagged with linting checks).

Typ. for both login input fields.